src/server/pause.js

```diff
@@ -0,0 +1,131 @@
+const test = window.d2lTest = {};
+test.pause = new Promise(r => test.start = r);
+
+const controls = `
+	<style>
+		#d2l-test-controls [hidden] {
+			display: none !important;
+		}
+
+		#d2l-test-controls #start, #run {
```

Comment:
Since these end up in the global namespace, I might suggest prefixing everything with `d2l-test-*` (i.e. `#d2l-test-run`, `#d2l-test-start`) just to avoid the crazy situation where someone uses an `id="start"` on one of their top-level components.

Reply:
I guess you mean to prevent them from accidentally selecting the controls? Not entirely opposed to that, but I think I'd rather convert the whole thing to a component after we get it merged.

Reply:
Yeah either a `getElementById` type collision or in this case this CSS would target their stuff and do weird things. But if the plan is to wrap it in a shadow root, then those concerns go away.

Reply:
Well this targets stuff inside `#d2l-test-controls` (except I missed `#run` here), but point taken.